dotcom-rendering/src/components/Pill.tsx

```diff
@@ -44,15 +49,20 @@ const pillPrefixStyles = css`
 	border-right: 1px solid ${palette('--pill-divider')};
 `;
 
-export const Pill = ({ content, prefix, icon, iconSide = 'left' }: Props) => {
+export const Pill = ({
+	content,
+	prefix,
+	icon,
+	iconSide = 'left',
+	iconSize = 'xsmall',
+}: Props) => {
 	const Icon = () =>
 		icon
 			? cloneElement(icon, {
```

Comment:
It's not a part of this PR, but are we expecting icon to be a certain type of React element (line 17), given it needs to have the `size` and `theme` props?